.github/workflows/release.yml

```diff
@@ -32,6 +21,8 @@ jobs:
           node-version: ${{ matrix.node-version }}
       - run: npm install
       - run: npm run lint
+      - run: docker-compose up -d
```

Comment:
We'll probably need to add the `env` field used in the test workflow here as well.